apps/web/src/components/ExperienceAccordion/ExperienceAccordionHeader.tsx

```diff
@@ -0,0 +1,102 @@
+
+import Accordion from "@gc-digital-talent/ui/src/components/Accordion";
+import { AccordionHeaderProps } from "@gc-digital-talent/ui/src/components/Accordion/Accordion";
+import { Link, Separator } from "@gc-digital-talent/ui";
+
+export interface ExperienceAccordionHeaderProps extends AccordionHeaderProps {
+  category?: React.ReactNode;
+  dateRange?: React.ReactNode;
+  editUrl?: string;
+}
+
+export const ExperienceAccordionHeader = ({
+  category,
+  dateRange,
+  editUrl,
+  children,
+  ...rest
+}: ExperienceAccordionHeaderProps) => {
+  const intl = useIntl();
+  return (
+    <Accordion.Header
+      data-h2-flex-grow="base(1)"
+      data-h2-display="base(flex)"
+      data-h2-flex-direction="base(row)"
+      data-h2-gap="base(x.5 0)"
+      {...rest}
+    >
+      <Accordion.Trigger>
+        <div
+          data-h2-flex-grow="base(1)"
+          data-h2-display="base(flex)"
+          data-h2-flex-direction="base(row)"
+          data-h2-gap="base(x.5 0)"
+        >
+          <div
+            data-h2-flex-grow="base(1)"
+            data-h2-display="base(flex)"
+            data-h2-flex-direction="base(column)"
+            data-h2-gap="base(x.5 0)"
+          >
+            <span data-h2-display="base(block)" data-h2-font-size="base(h6, 1)">
+              {children}
+            </span>
+            <div
+              className="Accordion__Subtitle"
+              data-h2-display="base(block)"
+              data-h2-font-size="base(copy)"
+              data-h2-margin="base(x.25 0 0 0)"
+            >
+              {category ? (
+                <span data-h2-color="base(primary)">{category}</span>
+              ) : null}
+              {category && dateRange ? (
+                <span
+                  data-h2-margin="base(0 x.5)"
+                  data-h2-color="base(gray.50)"
+                >
+                  |
+                </span>
+              ) : null}
+              {dateRange ? <span>{dateRange}</span> : null}
+            </div>
+          </div>
+        </div>
+      </Accordion.Trigger>
+      {editUrl ? (
+        <>
+          <Separator
+            orientation="vertical"
+            decorative
+            data-h2-background-color="base(gray.50)"
+            data-h2-margin="base(x1, 0)"
+            data-h2-height="base(unset)"
+          />
+          <div
+            data-h2-margin="base(x1)"
+            data-h2-display="base(flex)"
+            data-h2-flex-direction="base(column)"
+            data-h2-justify-content="base(center)"
+          >
+            <Link
+              href={editUrl}
+              data-h2-font-size="base(h6, 1)"
+              data-h2-color="base(primary.darker)"
+              data-h2-font-weight="base(700)"
+            >
+              {intl.formatMessage({
+                defaultMessage: "Edit",
+                id: "MdWZY5",
+                description: "Edit experience link label",
+              })}
+            </Link>
```

Comment:
We will need to add some more context to these edit links so when AT users use the list links functionality on this page they don't just hear a long list of "edit, edit, edit" for every experience they have listed. `Edit {title}` should be sufficient.

Reply:
Are you thinking hidden content?  Like `Edit<hidden> {title}</hidden>`?

Reply:
Yeah, I forgot how long some of our experience titles are 😅 . Hidden text makes sense here 👍

Reply:
It's a good catch.  Added in [add context to edit links](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6457/commits/bbea09bbd27511ebf0273422a8d40ce15c11092b).